### DISCUSSION TITLE
Welcome to booster Discussions!

### DISCUSSION THREAD

**Discussion by anatol** [anatol/booster#125] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by v-byte-cpu** [comment]
@anatol I would like to thank you from the bottom of my heart for such a wonderful project! I was using dracut before, and after another system upgrade I encountered an unpleasant booting problem. So I decided to look for modern initramfs alternatives and came across the booster project. I got luks + lvm working right out of the box! Especially nice is that the project is written in the go language. I wish you all the best for your life :heart: :pray:

**Reply by anatol** [reply]
I am glad that you found it useful!

Please keep using booster and provide feedback if there are any issues that can be improved.

**Reply by v-byte-cpu** [reply]
sure, I have already switched all my linux workstations to booster.
